app/assets/stylesheets/layout_components/reveal-modal.scss

```diff
@@ -20,9 +20,10 @@ dialog .close-reveal-modal{
     font-weight: normal;
   }
 
-  @include breakpoint(medium-up){
+  @include breakpoint(medium){
     &--small-window{
-      @extend .reveal-modal;
+      position: relative;
```

Comment:
FYI: I was extending the styles from reveal modal but when I updated the breakpoint to just be medium it gave me an error saying I couldn't extend .reveal-modal from within the media query. So I had to just apply the styles.